base.cfg

```diff
@@ -63,6 +63,8 @@ recipe = zc.recipe.testrunner
 eggs = ${instance:eggs}
 initialization =
     os.environ['TZ'] = 'UTC'
+    import time
```

Comment:
@wesleybl thanks for your PR! It would help to add a line or two of explanation here why we added this.

Reply:
Done.